example/src/components/CallControls.tsx

```diff
@@ -39,38 +39,28 @@ export const Controls = ({ disconnect }: { disconnect: () => void }) => {
 
   return (
     <View style={styles.iconsContainer}>
-      {!isMicrophoneOn ? (
+      <View style={styles.iconInRow}>
         <InCallButton
-          iconName="Microphone-off"
-          onPress={toggleMicrophoneAndUpdateMetadata}
-        />
-      ) : (
-        <InCallButton
-          iconName="Microphone"
-          onPress={toggleMicrophoneAndUpdateMetadata}
-        />
-      )}
-      {!isCameraOn ? (
-        <InCallButton
-          iconName="Cam-disabled"
+          iconName={!isCameraOn ? 'Cam-disabled' : 'Cam'}
           onPress={toggleCameraAndUpdateMetadata}
         />
-      ) : (
-        <InCallButton iconName="Cam" onPress={toggleCameraAndUpdateMetadata} />
-      )}
-      <InCallButton type="disconnect" iconName="Hangup" onPress={disconnect} />
-
-      {isScreencastOn ? (
+      </View>
+      <View style={styles.iconInRow}>
         <InCallButton
-          iconName="Screenshare"
-          onPress={toggleScreencastAndUpdateMetadata}
+          iconName={!isMicrophoneOn ? 'Microphone-off' : 'Microphone'}
+          onPress={toggleMicrophoneAndUpdateMetadata}
         />
-      ) : (
+      </View>
+      <View style={styles.iconInRow}>
         <InCallButton
-          iconName="Screen-off"
+          iconName={!isScreencastOn ? 'Screenshare' : 'Screen-off'}
           onPress={toggleScreencastAndUpdateMetadata}
         />
-      )}
+      </View>
+      <View style={styles.iconInRow}>
+        <InCallButton iconName="Menu-vertical" onPress={() => {}} />
```

Comment:
we don't need this button for now

Reply:
Ok, I'll remove it, it was part of the design so I added it.